### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [HalfSweet/Blog#21] [comment]
# about/

自我介绍 我是半糖，19岁，事学生 爱好电子，精通(看过)多种单片机使用方法以及编程语言 最喜欢的屏幕是墨水屏 计划学习更多有关ee和cs的内容，并希望今后从事类似的工作 会的东西 编程语言 略懂C/CPP语言 TODO 单片机 基于Arduino平台的AVR系列 基于Arduino/ESP-IDF/Micropython的ESP32系列 （其实是个连idf都不会装的菜鸡） 基于HAL库的STM32系列 8051/80251（特指STC32） CCS的安装与卸载

https://cf.blog.halfsweet.cn/about/

**Comment by CaiSong123** [comment]
大佬幼年期
